getting-started/README.adoc

```diff
@@ -170,6 +171,60 @@ ifndef::qs[]
 . Verify that your service account was successfully created in the *Streams for Apache Kafka* > *Service Accounts* page.
 endif::[]
 
+[id="proc-setting-service-account-permissions_{context}"]
+== Setting permissions for a service account in a Kafka instance in {product}
+
+{product} uses Access Control Lists (ACLs) provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create. When you create a service account to connect to a Kafka instance, you must also set the appropriate level of permissions for that new account in the ACL of the instance.
```

Comment:
Will add link to doc here to make in clear. See comment below listing our options

Reply:
@wtrocki Agreed, already did so in the access mgmt procedure and other sections there, pointing to Kafka for more details. I've just added a Kafka link here as well. Thanks.